app/locales/en/translations.js

```diff
@@ -344,6 +344,25 @@ export default {
             my_projects: `My OSF Projects`,
             headline: `On the OSF`,
         },
+        'preprint-status-banner': {
+            message: {
+                base: `{{name}} uses {{reviewsWorkflow}}, therefore this preprint is`,
```

Comment:
I understand why you split these messages into `base` and the rest, but I think that's over-optimizing. If we ever want to change some of the text (or translate into another language), it might require reworking the logic rather than just updating the translation file. I think it'd be better to be more verbose/repetitive and list the full sentences with all placeholders in each.